pass/provider.go

```diff
@@ -48,6 +56,8 @@ func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 	os.Setenv("PASSWORD_STORE_DIR", d.Get("store_dir").(string))
 
 	ctx := context.Background()
+	ctx = gpg.WithAlwaysTrust(ctx, true)
+	ctx = sub.WithCheckRecipients(ctx, false)
```

Comment:
Why not check `gpg.IsAlwaysTrust()` as it is done in https://github.com/gopasspw/gopass/blob/master/context.go#L24 ?

Reply:
Because we explicitly set it to true at the line above? I guessed this check was a remaining of the past due to someone setting always trust to true and not cleaning up afterward. But maybe I'm wrong here.

Reply:
It looks like both lines were added at the same time https://github.com/gopasspw/gopass/commit/e8ebcb44#diff-05a3aa0ab9e5687ff167e0ef3ae4c5a6R21-R46

Reply:
The lines were here before this commit https://github.com/gopasspw/gopass/commit/e8ebcb44#diff-7ddfb3e035b42cd70649cc33393fe32cL1083, we should look earlier in the history.